### ISSUE TITLE
Aria labels should show a value processed by the formatLabel prop and not the actual values

### ISSUE THREAD

**Issue by mradambeck** [davidchin/react-input-range#189]
For proper accessibility, aria attributes should be able to show the same content visually displayed in the label. Currently, they show the actual value of the slider. For example, if you are doing a percentage, you may have the value be `0.1` displayed as `10%`. There needs to be an option to hav e this same value placed into the `aria-valuemin`, `aria-valuemax` and `aria-valuenow` attributes.
